components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -23,6 +23,7 @@
 static NSString *const kReuseIdentifier = @"BaseCell";
 static const CGFloat kActionImageAlpha = (CGFloat)0.6;
 static const CGFloat kActionTextAlpha = (CGFloat)0.87;
+static const CGFloat kDividerDefaultOpacity = (CGFloat)0.12;
```

Comment:
Nit: Please name this `kDividerDefaultAlpha` to match the names of similar constants above.

Reply:
Updated, good catch!